specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2021-03-01-preview/mfe.json

```diff
@@ -0,0 +1,9458 @@
+              "description": "Azure Resource Manager resource envelope.",
+              "allOf": [
+                {
+                  "$ref": "#/definitions/BatchJobResource"
+                }
+              ],
+              "x-ms-azure-resource": true
+            }
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "machineLearningServices.json#/definitions/MachineLearningServiceError"
+            }
+          },
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/BatchJobResource"
+            }
+          },
+          "201": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/BatchJobResource"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "CreateOrUpdate Batch Job Endpoint.": {
+            "$ref": "./examples/BatchJobEndpoint/createOrUpdate.json"
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "BatchJob"
+        ],
+        "summary": "Lists batch inference endpoint jobs in this endpoint.",
+        "operationId": "BatchJobEndpoints_List",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "endpointName",
+            "description": "Name of endpoint",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/PaginationParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/WorkspaceNameParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "machineLearningServices.json#/definitions/MachineLearningServiceError"
+            }
+          },
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/BatchJobResourceArmPaginatedResult"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List Batch Job Endpoint.": {
+            "$ref": "./examples/BatchJobEndpoint/list.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/batchEndpoints/{endpointName}/{id}": {
```

Comment:
This URI is missting a segment between {endpointName}/{id}

Reply:
removed batchJob, as it was cut from PuP today :)